### ISSUE TITLE
Possible bug with weighted_path_loss

### ISSUE THREAD

**Issue by amorgun** [rosinality/stylegan2-pytorch#187]
This line looks weird: https://github.com/rosinality/stylegan2-pytorch/blob/master/train.py#L253
 Why would you need to multiply the term by zero?

**Comment by rosinality** [comment]
It is to trigger gradient all_reduce hook of the distributed data parallel. It will be not triggered when you use backward with the loss computed solely using gradients. (autograd.grad) Maybe it is changed in the recent versions of pytorch.
